specs/interop/predeploys.md

```diff
@@ -237,33 +238,55 @@ getters.
 |----------|----------------------------------------------|
 | Address  | `0x4200000000000000000000000000000000000015` |
 
-The `L1Block` contract is updated to include the set of allowed chains. The L1 Attributes transaction
-sets the set of allowed chains. The `L1Block` contract MUST provide a public getter to check if a particular
-chain is in the dependency set called `isInDependencySet(uint256)`. This function MUST return true when
-the chain's chain id is passed in as an argument.
-
-The `setL1BlockValuesInterop()` function MUST be called on every block after the interop upgrade block.
-The interop upgrade block itself MUST include a call to `setL1BlockValuesEcotone`.
-
-### L1Attributes
-
-The L1 Atrributes transaction is updated to include the dependency set. Since the dependency set is dynamically sized,
-a `uint8` "interopSetSize" parameter prefixes tightly packed `uint256` values that represent each chain id.
-
-| Input arg         | Type                     | Calldata bytes          | Segment |
-|-------------------|--------------------------|-------------------------|---------|
-| {0x760ee04d}      | bytes4                   | 0-3                     | n/a     |
-| baseFeeScalar     | uint32                   | 4-7                     | 1       |
-| blobBaseFeeScalar | uint32                   | 8-11                    |         |
-| sequenceNumber    | uint64                   | 12-19                   |         |
-| l1BlockTimestamp  | uint64                   | 20-27                   |         |
-| l1BlockNumber     | uint64                   | 28-35                   |         |
-| basefee           | uint256                  | 36-67                   | 2       |
-| blobBaseFee       | uint256                  | 68-99                   | 3       |
-| l1BlockHash       | bytes32                  | 100-131                 | 4       |
-| batcherHash       | bytes32                  | 132-163                 | 5       |
-| interopSetSize    | uint8                    | 164-165                 | 6       |
-| chainIds          | uint256\[interopSetSize] | 165-(32*interopSetSize) | 6+      |
+### Static Configuration
+
+The `L1Block` contract MUST include method `setConfig(ConfigType, bytes)` for setting the system's static values, which
```

Comment:
This is missing details on auth - can anybody call it? I think it should only be the depositor account

Reply:
good catch!